crates/protocol/interop/src/depset.rs

```diff
@@ -0,0 +1,225 @@
+use crate::MESSAGE_EXPIRY_WINDOW;
+use alloy_primitives::ChainId;
+use kona_registry::HashMap;
+
+/// Configuration for a dependency of a chain
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "serde", serde(rename_all = "camelCase"))]
+pub struct ChainDependency {
+    /// chain_index is the unique short identifier for this chain
+    pub chain_index: u32,
+
+    /// activation_time is the timestamp (in seconds) when the chain becomes part of the dependency
+    /// set. This is the minimum timestamp of the inclusion of an executing message that can be
+    /// considered valid.
```

Comment:
```suggestion
    /// Timestamp (in seconds) when the chain becomes part of the dependency
    /// set. This is the minimum timestamp of the inclusion of an executing message that can be
    /// considered valid.
```
try to avoid using field names in docs since they may change. if they are publicly accessible fields, then its ok to use a doc link e.g. (activation_time)(Self::activation_time).